testkit/src/main/scala/org/bitcoins/testkit/eclair/rpc/EclairRpcTestUtil.scala

```diff
@@ -255,7 +263,7 @@ trait EclairRpcTestUtil extends BitcoinSLogger {
       failFast: Boolean = true)(implicit system: ActorSystem): Future[Unit] = {
     awaitUntilPaymentStatus(client,
                             paymentId,
-                            PaymentStatus.SUCCEEDED,
+                            "sent",
```

Comment:
I don't like removing types here and replacing them with `String`. Now we will never get compiler errors if these things are changed again. Why did you remove the `PaymentStatus` here?

Reply:
Because there's no such entity in the RPC responses anymore

Reply:
So do we need to change the `awaitUntilPaymentStatus` interface to account for this then? 

Could we keep the `PaymentStatus` stuff, but make it internal to `EclairRpcTestUtil` and make it `private`?

Reply:
Those string  literals are only used locally inside `EclairRpcTestUtil.awaitUntilPaymentStatus` I cant see any problem with doing it. I can replace them with `Int`s.

Reply:
Fair enough, as long as we don't expose `awaitUntilPaymentStatus()`. I don't like using raw Strings as parameters as it is very hard to update  as we don't get compiler errors if we add a new String that represents a state. For instance if we had to segregate "failed" into "failed-onchain"/"failed-routing". We need to remember to update it, rather than having the compiler work for us and remind us to update it.